src/components/Drawer/index.js

```diff
@@ -85,6 +71,8 @@ export default function Drawer(props) {
         }
     }, [drawerState, isOpen, onOpened]);
 
+    useScrollLock(isOpen);
```

Comment:
last example in drawer (https://react-rainbow.io/#!/Drawer/7) is not working properly, when open drawer then select birthday input which open a modal, when click outside it closes both the modal and the drawer. It must only close the modal. (Before fix this issue let’s add an integration test to this case)
Maybe it need to be used in Modal in order to count the locks, but Modal is class then I think we should leave Drawer as before for now and use the hook only in the internal dropdown for now